web-gui/src/app/app.module.ts

```diff
@@ -100,8 +100,8 @@ import {CloseTicketDialogComponent} from './components/courses/detail-ticket/clo
 export class ApiURIHttpInterceptor implements HttpInterceptor {
   public intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     const clonedRequest: HttpRequest<any> = req.clone({
-      url: (req.url.search('localhost') >= 0) ? req.url : 'https://localhost'  + req.url // 'https://fk-server.mni.thm.de'
-      // url: 'https://feedback.mni.thm.de/'  + req.url // 'https://fk-server.mni.thm.de'
+      // url: (req.url.search('localhost') >= 0) ? req.url : 'https://localhost'  + req.url // 'https://fk-server.mni.thm.de'
+      url: 'https://feedback.mni.thm.de/'  + req.url // 'https://fk-server.mni.thm.de'
```

Comment:
Hey, at a fist glance everything looks fine to me.
Can you revert your changes made to redirect your call to the production server, please?